framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -3148,6 +3148,15 @@ private void stubDebug(String warning) {
      * #processField}.
      */
     private class AjavaAnnotationCollectorVisitor extends DefaultJointVisitor {
+        @Override
+        public Void visitCompilationUnit(CompilationUnitTree javacTree, Node javaParserNode) {
+            CompilationUnit node = castNode(CompilationUnit.class, javaParserNode, javacTree);
+            processCompilationUnit(javacTree, node);
```

Comment:
Please add a comment what this override is doing - or not doing - compared to the super method.